packages/components/src/formControls/CheckboxesList/CheckboxesList.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback } from 'react';
+
+import BaseInputsList from '@ids-internal/partials/BaseInputsList';
+import CheckboxField from '../CheckboxField';
+import withStateValue from '@ids-internal/hoc/withStateValue';
```

Comment:
is the order correct?
```suggestion
import withStateValue from '@ids-internal/hoc/withStateValue';

import CheckboxField from '../CheckboxField';
```